src/workerd/api/global-scope.c++

```diff
@@ -190,7 +191,8 @@ kj::Promise<DeferredProxy<void>> ServiceWorkerGlobalScope::request(kj::HttpMetho
 
   auto jsRequest = jsg::alloc<Request>(method, url, Request::Redirect::MANUAL, kj::mv(jsHeaders),
       jsg::alloc<Fetcher>(IoContext::NEXT_CLIENT_CHANNEL, Fetcher::RequiresHostAndProtocol::YES),
-      kj::none /** AbortSignal **/, kj::mv(cf), kj::mv(body));
+      /* signal */ kj::none, kj::mv(cf), kj::mv(body), Request::CacheMode::NONE,
+      kj::mv(abortSignal));
```

Comment:
Given that there's already an `AbortSignal` argument to the request constructor, some comments here explaining the additional argument would be helpful